robot_smach_states/src/robot_smach_states/navigation/navigation.py

```diff
@@ -356,7 +356,7 @@ def navigate_with_constraints(robot=None, constraint="x^2+y^2<1", frame="/map"):
     nwc = NavigateWithConstraints(robot, p, o)
     nwc.execute()
 
-class Turn(smach.State):
+class Turn(smach.State):  # ToDo: remove: this is just a force drive
```

Comment:
This smach state is currently in use by the spr challenge (outdated) and the restaurant challenge (lines 76,103,143 and 174). I suggest changing this first in order not to break the challenge (I know this is not what we discussed, but if breaking a challenge can easily be circumvented why not do that)

Reply:
Indeed, keep it.
```suggestion
class Turn(smach.State):
```

Reply:
No we are not keeping it, it should just be changed in the relevant places!

Reply:
All the challenges should check the construction of the challenge, see #900 for RIPS. Then it will easily show where Turn should be replaced by ForceDrive.